### PR TITLE
[th2-1515] removed dependency on Bintray

### PR DESCRIPTION
**Pull request by Maximus905**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -48,10 +53,10 @@ jar {
 }
 
 dependencies {
-    api platform('com.exactpro.th2:bom:2.10.1')
+    api platform('com.exactpro.th2:bom:3.0.0')
 
-    implementation 'com.exactpro.th2:common:2.16.0'
-    implementation 'com.exactpro.th2:store-common:2.7.0'
+    implementation 'com.exactpro.th2:common:2.17.1'
```
Comment (OptimumOpium) [inline build.gradle:58]: Please, update common to 2.17.2

**Review by OptimumOpium — CHANGES_REQUESTED** [review]

File: build.gradle [hunk 2]
```diff
@@ -22,10 +22,15 @@ repositories {
...
-    }
+	maven {
+		name 'Sonatype_snapshots'
+		url 'https://s01.oss.sonatype.org/content/repositories/snapshots/'
+	}
+
+	maven {
+		name 'Sonatype_releases'
+		url 'https://s01.oss.sonatype.org/content/repositories/releases/'
+	}
```
Comment (OptimumOpium) [inline build.gradle:33]: Please, format repository declaration

File: build.gradle (hunk 1 above)
Comment (Maximus905) [reply build.gradle]: done

File: build.gradle (hunk 2 above)
Comment (Maximus905) [reply build.gradle]: done

**Review by OptimumOpium — APPROVED** [review]

**Pull request merged**